.env

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
docker-compose will use this file to load defaults and I am also loading these in index.js as defaults as well.